src/compressed_tensors/compressors/model_compressors/model_compressor.py

```diff
@@ -103,7 +103,9 @@ def from_pretrained(
         :return: compressor for the configs, or None if model is not compressed
         """
         config = AutoConfig.from_pretrained(pretrained_model_name_or_path, **kwargs)
-        compression_config = getattr(config, COMPRESSION_CONFIG_NAME, None)
+        compression_config = getattr(config, COMPRESSION_CONFIG_NAME, None) or getattr(
+            config, QUANTIZATION_CONFIG_NAME, None
```

Comment:
I actually think this should only be QUANTIZATION_CONFIG_NAME  - it doesn't seem like in general that we're supporting loading `compression_config` anymore?

Reply:
Yeah I dont think so, i just kept it for backwards compatibility.
Let me remove and check tests so we can find out if remove would be appropriate. 

Doesnt hurt to keep, but have to remove eventually

Reply:
But have we kept backwards compatibility after removing SparseAutoModelForCausalLM pathway?

I don't think we did. 
In which case, that case is now irrelevant and should be removed.

Reply:
This is not for backwards compatibility depending on what happened with SparseAutoModelForCausalLM. 

I'm all for it to remove -- the reason why i left is because it loads from the config. 
If the loaded model has config of compression config, then it wont load. 

We can assume that models being loaded now all have quantization_config -- we need to get rid of it somepoint anyways

Reply:
Yeah its removed now, along with the COMPRESSION_CONFIG gloabl var